dipy/reconst/dti.py

```diff
@@ -82,6 +82,61 @@ def fractional_anisotropy(evals, axis=-1):
 
     return fa
 
+def geodesic_anisotropy(evals, axis=-1):
+    r"""
+    Geodesic anisotropy (FA) of a diffusion tensor.
+
+    Parameters
+    ----------
+    evals : array-like
+        Eigenvalues of a diffusion tensor.
+    axis : int
+        Axis of `evals` which contains 3 eigenvalues.
+
+    Returns
+    -------
+    ga : array
+        Calculated GA. In the range [0, +infty)
+
+    Notes
+    --------
+    GA is calculated using the following equation [1]_ and [2]_:
+
+    .. math::
+
+        GA = \sqrt{\sum_{i=1}^3 \log^2{\left ( \lambda_i/<\mathbf{D}> \right )}, 
+        \quad \textrm{where} \quad <\mathbf{D}> = \frac{\lambda_1+\lambda_2+\lambda_3}{3}
+
+
+    References
+    ----------
+
+    .. [1] P. G. Batchelor, M. Moakher, D. Atkinson, F. Calamante, A. Connelly, 
+        "A rigorous framework for diffusion tensor calculus", Magnetic Resonance 
+        in Medicine, vol. 53, pp. 221-225, 2005.
+    """
+
+    evals = _roll_evals(evals, axis)
+
+    all_zero = (evals == 0).all(axis=0)
+    ev1, ev2, ev3 = evals
+
+    md = (ev1 + ev2 + ev3)/3
```

Comment:
No. md = (l1+l2+l3)/3 whereas detD^(1/3) = (l1*l2*l3)^(1/3). I have realized that there are conflicting definitions of GA in papers. See [1] vs [2]. And other groups compute matrix_log of the whole tensor to generalize FA to log-Euclidean space.

[1] P. G. Batchelor, M. Moakher, D. Atkinson, F. Calamante, A. Connelly, 
        "A rigorous framework for diffusion tensor calculus", Magnetic Resonance 
        in Medicine, vol. 53, pp. 221-225, 2005.

[2] M. Morgado Correia, V.F.J. Newcombe, Guy B. Williams. "Contrast-to-noise 
       ratios for indices of anisotropy obtained from diffusion MRI: A study with 
       standard clinical b-values at 3 T", NeuroImage, vol. 57, pp. 1103-1115, 2011. 
 
I will keep looking and asking around... I have a feeling [1] should be trusted... 
Anyone knows in the Dipy community?